packages/wonder-blocks-modal/components/modal-dialog.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {View} from "wonder-blocks-core";
+
+import typeof ModalContentPane from "./modal-content-pane.js";
+
+export default class ModalDialog extends React.Component<{
+    children: React.ChildrenArray<React.Element<
+        ModalContentPane,
+    > | null | void>,
+    style?: any,
+}> {
+    render() {
+        const {style, children} = this.props;
+        return (
+            <View
+                style={[styles.wrapper, style]}
+                role="dialog"
+                aria-labelledby="wb-modal-title"
+            >
+                {children}
+            </View>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    wrapper: {
+        display: "flex",
+        flexDirection: "row",
```

Comment:
This might be a little surprising to someone who's doing build-your-own modal.  I think we should keep this `"column"` by default and override it in `two-column-modal.jsx`.

Reply:
I... disagree. Both TwoColumnModal and StandardModal need the flexDirection of row. I can't think of a situation where you would want two ModalContentPanes stacked on top of each other (which is what column would do). Not a single design has that, so I feel like having it default to row makes more sense.

Reply:
I was thinking of a situation where a person creates a modal using `ModalDialog` and fills it in using their own custom components.  Since that should be discouraged and probably pretty edge case-y I guess we can keep this as is.